### PR TITLE
feat: added admin module

### PR DESCRIPTION
**Pull request by NilanjanDaw**
bug: need to add authentication

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,8 +4,50 @@ var cookieParser = require('cookie-parser');
 var fs = require('fs');
 var logger = require('morgan');
 const webpush = require('web-push');
-
 var indexRouter = require('./routes/index');
+const AdminBro = require('admin-bro')
+const AdminBroSequelize = require('admin-bro-sequelizejs')
+const AdminBroExpress = require('admin-bro-expressjs')
+const bcrypt = require('bcrypt')
```
Comment (oyeb) [inline app.js]: This is a nitpick: recommend using [agon2](https://www.npmjs.com/package/argon2)

File: models/user.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    username: DataTypes.STRING,
+    email: DataTypes.STRING,
```
Comment (oyeb) [inline models/user.js]: Little bit validation:
```js
    email: {
      type: DataTypes.STRING,
      allowNull: false,
      validate: {
        isEmail: true
      }
```

File: package.json [hunk 3]
```diff
@@ -9,11 +9,17 @@
...
+    "admin-bro": "^2.2.3",
+    "admin-bro-expressjs": "^2.0.1",
+    "admin-bro-sequelizejs": "^0.4.0",
+    "bcrypt": "^4.0.1",
+    "cookie-parser": "^1.4.5",
     "debug": "~2.6.9",
     "dotenv": "^8.2.0",
     "ejs": "^3.0.1",
     "express": "~4.16.1",
+    "express-formidable": "^1.2.0",
```
Comment (oyeb) [inline package.json:21]: Doesn't seem to be used.

**Review by oyeb — CHANGES_REQUESTED** [review]

File: app.js [hunk 4]
```diff
@@ -4,8 +4,50 @@ var cookieParser = require('cookie-parser');
...
+AdminBro.registerAdapter(AdminBroSequelize)
+const models = require('./models')
+let adminBro = new AdminBro({
+    databases: [models],
+    rootPath: '/admin',
+    branding: {
+        companyName: 'PPE Help'
```
Comment (shinjanxp) [inline app.js]: **COVID-19 PPE Tracker** please

File: package.json (hunk 3 above)
Comment (NilanjanDaw) [reply package.json:21]: Formidable is used by the admin panel as an external dependency

**Review by shinjanxp — APPROVED** [review]
Looks good to me. @oyeb shall we merge?

**Review by oyeb — APPROVED** [review]
:+1:

**Pull request merged**